src/plugins/explore/public/components/query_panel/utils/typeDetection/constants.ts

```diff
@@ -0,0 +1,6 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export const KnownFields = ['status_code', 'category', 'user_id', 'timestamp', 'error'];
```

Comment:
is this for testing purposes?

we wont know if these fields are existing or not